docs/exploring-es-data.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[exploring-es-data]]
+= Exploring data in Elasticsearch
+
+[partintro]
+--
+By default Elastic APM data are stored in different indices, 
+in the format of `apm-[version]-[type]-[date]`.
+
+This format makes it easy to query for all Elastic APM data at once by using `apm-*`.
+
+It also allows for querying documents that have been collected with a specific APM Server version, e.g. `apm-6.3.0*`.
```

Comment:
i think you should either remove the `e.g. apm-6.3.0*` bit or use a variable pulling the current version